engines/dradis-api/app/controllers/dradis/ce/api/v1/evidence_controller.rb

```diff
@@ -8,6 +8,7 @@ class EvidenceController < Dradis::CE::API::APIController
 
       def index
         @evidence = @node.evidence.all.order('updated_at desc')
```

Comment:
Seeing `.all.` is missing from Issues and Nodes, but pressent in Evidence and Notes, what gives?

Also, it is my understanding that `.all.`, in recent versions of Rails, has turned into a scope and it no longer transforms the relation into an array. Can you confirm that?

Reply:
> Also, it is my understanding that .all., in recent versions of Rails, has turned into a scope and it no longer transforms the relation into an array.

At the moment it returns `AssociationRelation` vs `CollectionProxy` when not using `.all`. In this use case, I believe keeping it as the `CollectionProxy` makes more sense, so I'll remove the `.all` call.